charts/rancher-monitoring/v0.3.1/Chart.yaml

```diff
@@ -9,7 +9,7 @@ sources:
 - https://github.com/coreos/prometheus-operator
 version: 0.3.1
 appVersion: 0.3.1
-kubeVersion: ">=1.16.0-0"
+kubeVersion: "< 1.21.0-0"
```

Comment:
have to drop the lower bound because of a bug in rancher:

- Helm_v2 does not accept `>=1.16.0-0 <1.22.0-0`
- Rancher does not accept `>=1.16.0-0, <1.22.0-0`

so I have to keep the upper bound only.